edx_proctoring/templates/timed_exam/footer.html

```diff
@@ -2,7 +2,7 @@
 <div class="footer-sequence">
   <h4> {% trans "Can I request additional time to complete my exam?" %} </h4>
     <p>{% blocktrans %}
-      If you have disabilities or are taking the exam in difficult conditions,
+      If you have disabilities,
```

Comment:
https://openedx.atlassian.net/browse/SOL-1826 has been filed to get translation working for this repo.